docs/content/grafana/README.md

```diff
@@ -21,12 +40,54 @@ To add your dashboards to Grafana, you will need to create a `GrafanaDashboard`
 ## Using the Dashboards
 
 Grafana dashboards can easily be exported and imported, either from the UI or from the HTTP API.
-Dashboards are exported in Grafana JSON format, and contain everything you need (layout, variables, styles, data sources, queries, etc) to import the dashboard at a later time.
+Dashboards are exported in Grafana `JSON` format, and contain everything you need (layout, variables, styles, data sources, queries, etc) to import the dashboard at a later time.
 
-You can import these dashboards in Grafana either by pasting the dashboard JSON text directly into the text area or directly uploading the JSON file. Make sure to add/connect your [data sources][8] to Grafana and pick what data source you want the dashboard to use.
+You can import these dashboards in Grafana either by pasting the dashboard `JSON` text directly into the text area or directly uploading the `JSON` file. Make sure to add/connect your [data sources][8] (official Grafana data source documentation) to Grafana and pick what data source you want the dashboard to use.
```

Comment:
> (official Grafana data source documentation)

Hmm...don't think this makes it any more clear, I would instead word it like this: 

> Make sure to add/connect your data sources to Grafana and pick what data source you want the dashboard to use. To learn more about this please visit the [Grafana docs on data sources](https://grafana.com/docs/grafana/latest/datasources/).